README.md

```diff
@@ -1,16 +1,15 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio - Jorum Odiemo
 
-> Description the project.
+> Creating a skeleton mobile website of my portfolio
+> 
 
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
+- HTML & CSS3
+- Flexbox and Grid
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] Please make sure to customise the README template, I can see that some unused sections were not cleared: 
<img width="1063" alt="Screenshot 2022-06-08 at 18 42 48" src="https://user-images.githubusercontent.com/77038610/172645467-dc2f4528-ac38-41e4-9317-be033a21edcf.png">

Please customize this section by adding proper steps to get started.